pkg/util/translate/single_namespace.go

```diff
@@ -104,8 +112,6 @@ func (s *singleNamespace) IsManagedCluster(obj runtime.Object) bool {
 	metaAccessor, err := meta.Accessor(obj)
 	if err != nil {
 		return false
-	} else if metaAccessor.GetLabels() == nil {
-		return false
 	}
```

Comment:
Don't we need a label nil check here?

Reply:
No, I didn't knew that as well, but this works:
```
var m map[string]bool = nil

fmt.Println(m["my-key"])
```